Packs/Cybereason/Integrations/Cybereason/Cybereason.py

```diff
@@ -401,7 +404,7 @@ def query_connections_command(client: Client, args: dict):
 
         return CommandResults(
             readable_output=tableToMarkdown(f'Cybereason Connections for: {filter_input}', outputs),
-            outputs_prefix='Cybereason.Connection',
+            outputs_prefix='Connection',
```

Comment:
I strongly advice against this change. Think about a case where a client is using both you product and another one that can search for connection. Then the client needs to be able to know what came from your integration and what came for another one.

Reply:
Sure Jawine.
That makes complete sense. We will add the Cybereason Prefix again.
Thanks for pointing it out.

Reply:
Fixed now